### PR TITLE
Clarify that lineDash has no effect in IE 10 and below

### PR DESCRIPTION
**Pull request by marcjansen**
See #5114

### CODE REVIEW

File: externs/olx.js [hunk 1]
```diff
@@ -6686,7 +6686,13 @@ olx.style.StrokeOptions.prototype.lineJoin;
 
 
 /**
- * Line dash pattern. Default is `undefined` (no dash).
+ * Line dash pattern. Default is `undefined` (no dash). Please note that
+ * Internet Explorer 10 [does not support][mdn] the `setLineDash` method on the
```
Comment (tsauerwein) [inline externs/olx.js]: 'Internet Explorer 10 and lower'?

Comment (marcjansen) [inline externs/olx.js]: Agreed

**Comment by marcjansen** [comment]
OK now @tsauerwein?

**Comment by tsauerwein** [comment]
Yes, thanks for updating! :+1:

File: externs/olx.js [hunk 2]
```diff
@@ -6686,7 +6686,13 @@ olx.style.StrokeOptions.prototype.lineJoin;
 
 
 /**
- * Line dash pattern. Default is `undefined` (no dash).
+ * Line dash pattern. Default is `undefined` (no dash). Please note that
+ * Internet Explorer 10 and lower [do not support][mdn] the `setLineDash`
+ * method on the `CanvasRenderingContext2D` and therefore this option will
+ * have no visual effect in these browsers.
+ *
+ * [mdn]: https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/setLineDash#Browser_compatibility
```
Comment (ahocevar) [inline externs/olx.js]: Have you checked if the Markdown parser can handle link refs like that?

Comment (marcjansen) [inline externs/olx.js]: It can:

image

**Comment by ahocevar** [comment]
Nice, please merge.

**Comment by marcjansen** [comment]
Cheers for the reviews!

**Pull request merged**